NitroxPatcher/Patches/Dynamic/Base_CopyFrom_Patch.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Harmony;
+using System.Reflection;
+using NitroxClient.MonoBehaviours;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    class Base_CopyFrom_Patch : NitroxPatch, IDynamicPatch
+    {
+        public readonly MethodInfo METHOD = typeof(Base).GetMethod(nameof(Base.CopyFrom), BindingFlags.Public | BindingFlags.Instance);
+
+        public static void Prefix(Base __instance, Base sourceBase)
+        {
+            NitroxEntity entity = sourceBase.GetComponent<NitroxEntity>();
+
+            if (sourceBase.GetComponent<BaseRoot>() == null)
```

Comment:
Don't use `== null` checks on Unity objects because they can be dead but not null.
```suggestion
            if (!sourceBase.GetComponent<BaseRoot>())
```